curve-ansible/deploy_curve.yml

```diff
@@ -127,19 +127,19 @@
     - { role: generate_config, template_name: chunkserver.conf, conf_path: "{{ chunkserver_conf_path }}",
                 tags: ["generate_config", "generage_chunkserver_conf"] }
     - { role: generate_config, template_name: client.conf, conf_path: "{{ chunkserver_client_config_path }}",
-                tags: ["generate_config", "generage_chunkserver_conf"] }
+                tags: ["generate_config", "generage_chunkserver_client_conf"] }
```

Comment:
Curve-ansible here change some tags. Do you check if any deploy document or wiki need change？

Reply:
> Curve-ansible here change some tags. Do you check if any deploy document or wiki need change？

yes, the doc use like "ansible-playbook deploy_curve.yml -i server.ini --tags generage_chunkserver_conf,generage_chunkserver_client_conf". This issue found in the process of expansion.